docs/pages/notes.md

```diff
@@ -78,6 +78,12 @@ We only officially support what we test.
 
 We test Node versions that are `Active LTS` and `Current`. For which versions of Node that equals you can refer to our tests or look here: https://nodejs.org/en/about/releases.
 
+### Supported Packages managers
+
+1. `Yarn` we can use yarn at any deployment platform except `Heroku` because Heroku starting `Pruning devDependencies` and yarn rebuild fresh packages this means that the generated code by the `prisma generate` command will delete. We can use `Yarn` with `Heroku` by adding a custom [Gentime Settings](/settings-gentime#usage) and change the output path.
```

Comment:
Is this the same guidance provided by Prisma Client?

Reply:
What about Yarn 2?

Reply:
> Prisma Client generates into the node_modules/.prisma folder, so it's not affected by Pruning devDependencies.

Then this is what we should do.

Reply:
@andrewicarlson and I talked about this option before, and we found that Prisma had an issue with `Yarn 2` https://github.com/prisma/prisma/issues/1439 and @matthewmueller recommends a workaround to a custom output path

Reply:
That is support for Yarn 2 with PnP, I don't have stats, but I suspect that's pretty niche part of the community? Support for Yarn 2 non-PnP and Yarn 1 would be the way to go I think.

I would also prefer to follow the lead of Prisma Client and do whatever they do. It is more consistent for users and if we just explicitly say that this is our policy we can ensure better compat for our users and their stack planning.

All NP users are P users so not being consistent with P doesn't make much sense IMO.

Reply:
It was my opinion to go with prisma client way in everything, but @andrewicarlson said that we could wait to see how prisma fixes this issue and go with their way.

@andrewicarlson what is your opinion in this? Can we go with the Prisma client?

Reply:
Like we talked about before, and like Jason reiterated above:

>I would also prefer to follow the lead of Prisma Client and do whatever they do (unless we have a technically superior path we can trail blaze).

We don't have a technically superior path identified at this time, so we should follow the Prisma Client's lead!